sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -29,12 +42,32 @@
  */
 @ServiceClient(builder = DigitalTwinsClientBuilder.class)
 public class DigitalTwinsAsyncClient {
-    private final ClientLogger logger = new ClientLogger(DigitalTwinsAsyncClient.class);
+    private static final ClientLogger logger = new ClientLogger(DigitalTwinsAsyncClient.class);
+    private static final ObjectMapper mapper = new ObjectMapper();
     private final DigitalTwinsServiceVersion serviceVersion;
     private final AzureDigitalTwinsAPI protocolLayer;
 
     DigitalTwinsAsyncClient(HttpPipeline pipeline, DigitalTwinsServiceVersion serviceVersion, String host) {
-        this.protocolLayer = new AzureDigitalTwinsAPIBuilder().host(host).pipeline(pipeline).buildClient();
+        final SimpleModule stringModule = new SimpleModule("String Serializer");
+        stringModule.addSerializer(new StdSerializer<String>(String.class, false){
+            @Override public void serialize(String value, JsonGenerator gen, SerializerProvider provider)
+                throws IOException {
+                if (isJSONValid(value)) {
+                    gen.writeRawValue(value);
+                } else {
+                    gen.writeString(value);
+                }
+            }
+        });
+
+        JacksonAdapter jacksonAdapter = new JacksonAdapter();
```

Comment:
Should this be created each time this method is called, or could it be cached on the class and reused?